docs/articles/nunit/release-notes/framework.md

```diff
@@ -6,6 +6,63 @@ uid: frameworkreleasenotes
 
 # Framework Release
 
+## NUnit 4.3.1 -  Dec 21. 2024
+
+This is a hotfix release to address a type safety issue that broke when some classes had implicit conversion operators.
+
+We had 3 bugs reported, they are all fixed by the same Pull Request.
+
+IN addition a small internal build issue was fixed.
+
+### Bug fixes
+
+* [4903](https://github.com/nunit/nunit/issues/4903) Strange null reference after updating to v4.3.0. Thanks to NUnit Team member [Manfred Brands](https://github.com/manfred-brands) for [PR 4899](https://github.com/nunit/nunit/pull/4899)
+* [4902](https://github.com/nunit/nunit/issues/4902) Regression in 4.3.0 comparing class that override equals and implicit string conversion. Thanks to NUnit Team member [Manfred Brands](https://github.com/manfred-brands) for [PR 4899](https://github.com/nunit/nunit/pull/4899)
+* [4898](https://github.com/nunit/nunit/issues/4898) Assert(..., Is.EqualTo(...)) issues since 4.3.0. Thanks to NUnit Team member [Manfred Brands](https://github.com/manfred-brands) for [PR 4899](https://github.com/nunit/nunit/pull/4899)
+
+### Internal fixes
+
+* [4806](https://github.com/nunit/nunit/issues/4806) Benchmarks fail to run. Thanks to NUnit Team member [Steven Weerdenburg](https://github.com/stevenaw) for [PR 4897](https://github.com/nunit/nunit/pull/4897)
+
+### Acknowledgements
+
+We want to express our heartfelt gratitude to everyone who has contributed to this release
+by reporting bugs, suggesting enhancements, and providing valuable feedback.
+Your efforts help make NUnit better for the entire community.
+
+A special thank you to the following reporters for identifying issues:
+
+<table>
```

Comment:
Doesn't have to hold up the PR at all, but I notice this is an HTML table rather than a markdown table. We should probably use markdown tables.

Reply:
I wanted it without header, since it is more of a 4 column list of names.  Didnt find any markdown way to do that.  But if have some tricks, please suggest :-)

Reply:
I think it makes most sense as a bulleted list. Table probably shouldn't be used just to visually separate list items.